content/applications/general/email_communication/email_domain.rst

```diff
@@ -1,163 +1,190 @@
-========================
-Sending emails with Odoo
-========================
+=======================
+Send an email with Odoo
+=======================
 
-Using your email domain in Odoo
-===============================
+Use an email domain in Odoo
+===========================
 
 Documents in Odoo (a CRM opportunity, a sales order, an invoice ...) have a discussion thread,
 called *chatter*.
 
-When you post a message in the chatter, this message is sent by email to the followers of the
-document. If a follower replies to the message, the reply updates the chatter, and Odoo relays the
-reply to the followers.
+When a database user posts a message in the chatter, this message is sent by email to the followers
+of the document. If a follower replies to the message, the reply updates the chatter, and Odoo
+relays the reply to the followers.
 
-Emails from your users to partners (customers, vendors) are sent from the email address of your
-users. Similarly, emails from partners to users are sent from the email address of the partners.
-This allows you to recognize at a glance who sent an email relayed by Odoo.
+Emails from the database users to partners (customers or vendors) are sent from the email addresses
+of the database users. Similarly, emails from partners to users are sent from the email addresses
+of the partners. That way, the database users can recognize who sent an email relayed by Odoo.
 
-If your database is hosted on our cloud (Odoo Online or Odoo.sh), it is not necessary to add an
-outgoing email server to send emails from your custom domain. You can enjoy this feature by using
-the default Odoo email server.
+If the Odoo database is hosted on the cloud (Odoo Online or Odoo.sh), it is not necessary to add an
+outgoing email server to send emails from a custom domain.
 
 .. important::
    The Odoo server is subject to a daily email limit to prevent abuse. The default limit is 200
-   emails sent per day for databases with an Enterprise subscription. This limit can be increased
-   under certain conditions. See our :doc:`FAQ <faq>` or contact support for more
+   emails sent per day for databases with an **Enterprise** subscription. This limit can be
+   increased under certain conditions. See the :doc:`FAQ <faq>` or contact support for more
    information.
 
-However, it is recommended that you configure your domain name to ensure that emails from your
-users reach your partners, rather than being considered spam.
+To ensure that emails from the database users reach the partners, instead of being considered spam,
+Odoo recommends configuring the domain name.
```

Comment:
I know what we mean when we say`partners` but could see this being confusing for readers.

```suggestion
To ensure that emails sent to and from the chatter reach their intended contacts, instead of being
considered spam, Odoo recommends configuring the domain name.
```